hudi-client/hudi-flink-client/src/main/java/org/apache/hudi/client/HoodieFlinkTableServiceClient.java

```diff
@@ -65,15 +70,16 @@ public class HoodieFlinkTableServiceClient<T> extends BaseHoodieTableServiceClie
 
   protected HoodieFlinkTableServiceClient(HoodieEngineContext context,
                                           HoodieWriteConfig clientConfig,
-                                          Option<EmbeddedTimelineService> timelineService) {
-    super(context, clientConfig, timelineService);
+                                          Option<EmbeddedTimelineService> timelineService,
```

Comment:
@danny0405 can you put together or have a diagram that shows the class dependencies in Flink.. i.e what code runs on Driver/Coordinator vs Executor. classes involved etc. 

I want to make sure this works for Flink easily as well

Reply:
The write to MDT all happens on the Driver/coordinator, the commit also happens on Driver for DT, the actual write happens on executors.